register-service-broker.sh

```diff
@@ -28,5 +28,25 @@ for SERVICE in $(echo "$SERVICES"); do
   # Must disable services prior to enabling, otherwise enable will fail if already exists
   # https://github.com/cloudfoundry/cli/issues/939
   cf disable-service-access "${ARGS[@]}"
-  cf enable-service-access "${ARGS[@]}"
+
+  # If SERVICE_ORGANIZATION_BLACKLIST, then expect service to be singular
+  if [ -n "${SERVICE_ORGANIZATION_BLACKLIST:-}" ]; then
+    cf_orgs_output="$(cf orgs)"
+    blacklist_array="(${SERVICE_ORGANIZATION_BLACKLIST})"
+    org_array=()
+    while read -r org_array_line; do
```

Comment:
I think this whole thing can be simplified to something like this if we make grep do all the work:

	cf disable-service-access "${ARGS[@]}"

	if [ -n "${SERVICE_ORGANIZATION_BLACKLIST:-}" ]; then
		for org in `cf orgs | grep -Fvxf <(echo $SERVICE_ORGANIZATION_BLACKLIST | tr " " "\n")`; do
			cf enable-service-access "${ARGS[@]}" -o ${org}
		done
	else
		cf enable-service-access "${ARGS[@]}"
	fi